friends/src/App.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from "react";
+import axios from "axios";
+import { Route, Redirect } from "react-router-dom";
+import { FriendsList } from "./components/FriendsList";
+import { NewFriendForm } from "./components/NewFriendForm";
+import uuid from "uuid";
+import "./App.css";
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      friends: [],
+      potentialNewFriend: {
+        id: "",
+        name: "",
+        age: "",
+        email: "",
+        color: "",
+        favefood: "",
+        quotation: ""
+      }
+    };
+  }
+
+  componentDidMount() {
+    axios
+      .get("http://localhost:5000/friends")
+      .then(res => {
+        this.setState({ friends: res.data });
+      })
+      .catch(err => {
+        this.setState({ error: err });
+      });
+  }
+
+  handleChanges = e => {
+    e.preventDefault();
+    let tempData = this.state.potentialNewFriend;
+    tempData[e.target.name] = e.target.value;
+    this.setState({
+      potentialNewFriend: tempData
+    });
+  };
+
+  addNewFriend = () => {
+    axios
```

Comment:
right now your form is following the default behavior of refreshing the page -- this clears the fields but is still undesired behavior, I'd `preventDefault` here